src/test/java/seedu/address/model/TutorsPetTest.java

```diff
@@ -242,13 +269,13 @@ public void equals() {
         tutorsPetDifferentStudents.resetData(getTypicalTutorsPet());
         Student toDeleteStudent = tutorsPetDifferentStudents.getStudentList().get(0);
         tutorsPetDifferentStudents.deleteStudent(toDeleteStudent);
-        assertTrue(tutorsPet.getModuleClassList().equals(tutorsPetDifferentStudents.getModuleClassList()));
+        assertFalse(tutorsPet.getModuleClassList().equals(tutorsPetDifferentStudents.getModuleClassList()));
```

Comment:
Is this change because if a student is deleted, then the associated student in ModuleClass is removed as well?

Reply:
In view of the change, I would recommend changing the test case to remove a student that was not previously part of any class. This way, we would only be changing the student list and would be a better representation of the test case.

Reply:
@junlong4321 yup, that is right. 
@dextertanyj agreed, will remove a student not in any classes.